openmdao/docs/theory_manual/scaling.rst

```diff
@@ -0,0 +1,91 @@
+.. _theory_scaling:
+
+*******************
+Scaling in OpenMDAO
+*******************
+
+In the :ref:`feature doc on scaling <scale_outputs_and_resids>`, you learned how you can specify multiplicative and additive scale factors
+that are applied to the output and residual variables to aid solver convergence. This theory document shows how scaling is handled internally.
+
+When OpenMDAO runs a model, all of the numerical data from connections is assembled into a set of vectors called the "inputs" and "outputs"
+vectors. When a component calls its "compute" function, it is passed a sub-vector that contains views to just the data relevant to that component.
+A similar-sized vector called "residuals" is also created to house implicit and cycle-induced residuals. Each vector can transform all of its data
+or a portion of its data into a scaled or unscaled state when requested. Doing so requires multiplication with a vector of scalers and
+addition with a vector of adders.  There is a small amount of cost to this, so OpenMDAO tries to scale/unscale only when necessary and only over
+as tight a scope as required.
+
+Scaling is used primarily as an aid to solver convergence (both linear and nonlinear). Whenver a user interacts with OpenMDAO, or whenever a component
+calls a user-written function like "compute", the vectors will always be in an unscaled state. The following tables summarize this:
+
+**outputs and residuals (linear and nonlinear) are in an unscaled state**
+
+ - Before run_model is called
+ - After run_model is finished
+ - During ExplicitComponent API calls compute, compute_partials
+ - During ImplicitComponent API calls apply_nonlinear, linearize
+ - During guess_nonlinear
+ - When a Driver runs the model or computes a total derivative
+
+**outputs and residuals (linear and nonlinear) are in a scaled state**
+
+ - At all points "in between" the cases listed above
```

Comment:
I don't want to move the summary, because there will (someday) be many more diagrams here, and this is the important stuff for the user to know.  I have re-written it some and removed the "in between" line item.